### PR TITLE
WS-42 (Design Database)

### PR DESCRIPTION
**Pull request by Elmax19**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,21 +18,45 @@
...
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-core</artifactId>
+			<version>4.4.2</version>
+		</dependency>
+
+		<dependency>
```
Comment (arramanenka) [inline pom.xml:55]: iirc, this should not be necessary, only the <plugin> config is used in your case

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
 		<version>2.6.0</version>
-		<relativePath/> <!-- lookup parent from repository -->
+		<relativePath/>
 	</parent>
 	<groupId>com.itechart.students_lab</groupId>
```
Comment (arramanenka) [inline pom.xml:11]: I would recommend against snake cases for groupId, feel free to just use com.itechart

File: src/main/resources/db/changelog/changeset/create-address-table.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="create-address" author="Elmax19">
+        <createTable tableName="address">
```
Comment (arramanenka) [inline src/main/resources/db/changelog/changeset/create-address-table.xml:7]: Just an FYI, no changes are required: you could have written this as a plain SQL, which arguably could be a bit more readable for people. But as long as you all agree on one syntax, its fine with me

File: src/main/resources/db/changelog/db.changelog-master.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <include file="src/main/resources/db/changelog/changeset/create-address-table.xml"/>
```
Comment (arramanenka) [inline src/main/resources/db/changelog/db.changelog-master.xml:6]: iirc, there was a way to specify files using relative paths. Could you please check that?

Comment (Elmax19) [reply src/main/resources/db/changelog/db.changelog-master.xml:6]: It writes, that file was nat found

File: src/main/resources/db/changelog/changeset/create-address-table.xml (hunk 3 above)
Comment (Elmax19) [reply src/main/resources/db/changelog/changeset/create-address-table.xml:7]: Using xml format was like advice from the lecturer from today's lecture

File: src/main/resources/db/changelog/db.changelog-master.xml (hunk 4 above)
Comment (arramanenka) [reply src/main/resources/db/changelog/db.changelog-master.xml:6]: try with relativeToChangelogFile="true" in include

File: src/main/resources/db/changelog/changeset/create-address-table.xml (hunk 3 above)
Comment (arramanenka) [reply src/main/resources/db/changelog/changeset/create-address-table.xml:7]: Alright, whatever works for you

**Review by arramanenka — APPROVED** [review]

**Pull request merged**